src/content/community/get-involved/open-research/index.md

```diff
@@ -0,0 +1,165 @@
+---
+title: Active areas of Ethereum research
+description: Explore different areas of open research and learn how to get involved.
+sidebar: true
+lang: en
+---
+
+# Ethereum open areas of research
+
+One of the primary strengths of Ethereum is that it is constantly being improved by an active research and engineering community. There are many enthusiastic, skilled people worldwide that would like to apply themselves to outstanding issues in Ethereum, but it is not always easy to find out what those issues are. This page aims to outline some of the key areas of active research as a rough guide to Ethereum's cutting edge.
+
+## General research resources
+
+Regardless of the specific topic, there is a wealth of information on Ethereum research to be found at [ethresear.ch](https://ethresear.ch) and the [Eth R&D discord channel](https://discord.gg/n7JxAeRu). These are the primary places where Ethereum researchers discuss the latest ideas and development opportunities.
+
+## Scaling and Performance
+
+### Layer 2
+
+There are now several Layer 2 protocols that scale Ethereum using different techniques for batching transactions and securing them on Ethereum layer 1. This is a very rapidly growing topic with a lot of research and development potential.
+
+### Bridges
+
+One particular area of Layer-2 that requires more research and development is safe and performant bridges. This includes bridges between various Layer-2s and bridges between Layer 1 and Layer 2.
+
+### Sharding
+
+Sharding Ethereums blockchain has long been part of the development roadmap. However, new scaling solutions such as "Danksharding" are currently taking center stage.
+
+### Hardware
+
+Running nodes on modest hardware is fundamental to keeping Ethereum decentralized. Therefore, active research into minimizing the hardware requirements to run nodes is an important area of research.
```

Comment:
Perhaps adding intro context on running a node? e.g. https://ethereum.org/en/developers/docs/nodes-and-clients/run-a-node/